env/servers.env.example

```diff
@@ -0,0 +1,8 @@
+DEV_EMAIL_ADDRESS=
+JWT_SECRET=
+SESSION_COOKIE_SECRET=
+
+CENTRAL_API_URL=
+ENTITY_API_URL=
+REPORT_API_URL=
+WEB_CONFIG_API_URL=
```

Comment:
Mind if we move these out into a separate `api-client.env` file? I'm gonna be using the api-client in the e2e testing script, so will need these. Could we also add variables for all the `AUTH_API_URL` and `DATA_TABLE_API_URL`? :pray: